notebooks/Deploy command medium.ipynb

```diff
@@ -5,7 +5,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Deploy cohere-gpt-medium Model Package from AWS Marketplace \n",
+    "# Deploy Cohere Generate Model - Command-Light Model Package from AWS Marketplace \n",
```

Comment:
On lines 150 & 153 can you also change `cohere-gpt-medium` to `cohere-command-light`?

And can you also add a ticket or a note somewhere that for the next release we should change the package name from `cohere-gpt-medium-v1-15-a0c02e4b286b3f90a7794030dad56d9e` to `cohere-command-light-...`